doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -867,66 +841,71 @@ Simple inductive types
    .. exn:: The conclusion of @type is not valid; it must be built from @ident.
 
       The conclusion of the type of the constructors must be the inductive type
-      :token:`ident` being defined (or :token:`ident` applied to arguments in
+      :n:`@ident` being defined (or :n:`@ident` applied to arguments in
       the case of annotated inductive types — cf. next section).
 
-   .. example::
+The following subsections show examples of simple inductive types, simple annotated
+inductive types, simple parametric inductive types and mutually inductive
+types.
 
-      The set of natural numbers is defined as:
+.. _simple-inductive-types:
 
-      .. coqtop:: all
+Simple inductive types
+~~~~~~~~~~~~~~~~~~~~~~
 
-         Inductive nat : Set :=
-         | O : nat
-         | S : nat -> nat.
+A simple inductive type belongs to a universe that is a simple :n:`sort`.
```

Comment:
What does it mean to be a "simple sort"?  Is that different from @sort?  If so, that should be defined--I can do that in the other PR.  And :n:\`sort\` should be :n:\`@sort`

Reply:
It's just a sort. I think it is written a simple sort as opposed to a complex type.